config/forms/2021_2022.json

```
@@ -2050,6 +2050,243 @@
             }
           }
         },
+        "household_needs": {
+          "label": "Household needs",
+          "depends_on": [
+            {
+              "setup": "completed"
+            }
+          ],
+          "pages": {
+            "armed_forces": {
+              "header": "",
+              "description": "",
+              "questions": {
+                "armedforces": {
+                  "header": "Does anybody in the household have any links to the UK armed forces?",
+                  "hint_text": "This excludes national service.<br> If there are several people in the household with links to the UK armed forces, you should answer for the regular. If there’s no regular, answer for the reserve. If there’s no reserve, answer for the spouse or civil partner.",
```

I wonder if we should use line breaks character rather than HTML in JSON form definition (i.e. `\n`)? In the mean time…

```suggestion
                  "hint_text": "This excludes national service.<br><br>If there are several people in the household with links to the UK armed forces, you should answer for the regular. If there’s no regular, answer for the reserve. If there’s no reserve, answer for the spouse or civil partner.",
```